### ISSUE TITLE
eldev lint does not ignore package description file

### ISSUE THREAD

**Issue by sirikid** [doublep/eldev#34]
```
eldev lint
oj-pkg.el:0: The first line should be of the form: ";;; package --- Summary"
oj-pkg.el:0: You should have a section marked ";;; Commentary:"
oj-pkg.el:2: You should have a section marked ";;; Code:"
oj-pkg.el:4: The footer should be: (provide ’oj-pkg)\n;;; oj-pkg.el ends here
Found 4 warnings in file ‘oj-pkg.el’
```

**Comment by doublep** [comment]
Fixed in 0.9. The file is now also not byte-compiled (even without `-*- no-byte-compile: t -*-`). However, the file must be named after the project, i.e. `PROJECT-pkg.el`.

**Comment by sirikid** [comment]
Wow, cool. I have already forgot about this issue. Thank you, Paul.
